task/testmap.py

```diff
@@ -85,18 +85,12 @@
     },
     'weldr/lorax': {
         'master': [
+            'fedora-32/lorax',
             'fedora-32/osbuild-composer',
         ],
         'rhel8-branch': [
-            'rhel-8-3',
-            'rhel-8-3/live-iso',
-            'rhel-8-3/qcow2',
-            'rhel-8-3/aws',
-            'rhel-8-3/azure',
-            'rhel-8-3/openstack',
-            'rhel-8-3/vmware',
-            'rhel-8-3/tar',
-            'rhel-8-3/ci',
+            'rhel-8-4/lorax',
```

Comment:
@bcl from what I can see lorax' `test/run` doesn't respect the `ci` scenari anymore but does respect a `lorax` scenario so I've added that both for master and rhel8-branch branches.